src/Service.GraphQLBuilder/Azure.DataApiBuilder.Service.GraphQLBuilder.csproj

```diff
@@ -6,6 +6,7 @@
     <Nullable>enable</Nullable>
     <OutputPath>$(BaseOutputPath)\engine</OutputPath>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <NoWarn>NU1603</NoWarn>
```

Comment:
nit: If we are adding this property to all projects, is it better to maintain the property group at root level similar to the `Directory.Packages.props`?

Reply:
This change is not inteded to be added as part of this PR. This is showing up in the changeset because the target branch was not  updated with the latest main whereas this branch is. 

I've updated the target branch with the latest changes from main. So, this change shouldn't come up anymore.